lib/python/qmk/cli/doctor.py

```diff
@@ -182,18 +195,17 @@ def check_udev_rules():
                     current_rules.add(line)
 
         # Check if the desired rules are among the currently present rules
-        for bootloader, rules in desired_rules.items():
+        for rules in desired_rules.items():
             # For caterina, check if ModemManager is running
-            if bootloader == "caterina":
-                if check_modem_manager():
-                    ok = False
-                    cli.log.warn("{bg_yellow}Detected ModemManager without the necessary udev rules. Please either disable it or set the appropriate udev rules if you are using a Pro Micro.")
+            if check_modem_manager():
+                ok = False
+                cli.log.warn("{bg_yellow}Detected ModemManager without the necessary udev rules. Please either disable it or set the appropriate udev rules if you are using a Pro Micro.")
             if not rules.issubset(current_rules):
-                deprecated_rule = deprecated_rules.get(bootloader)
-                if deprecated_rule and deprecated_rule.issubset(current_rules):
-                    cli.log.warn("{bg_yellow}Found old, deprecated udev rules for '%s' boards. The new rules on https://docs.qmk.fm/#/faq_build?id=linux-udev-rules offer better security with the same functionality.", bootloader)
-                else:
-                    cli.log.warn("{bg_yellow}Missing udev rules for '%s' boards. You'll need to use `sudo` in order to flash them.", bootloader)
+                for bootloader, rule in deprecated_rules.items():
+                    if rule and rule.issubset(current_rules):
+                        cli.log.warn("{bg_yellow}Found old, deprecated udev rules for '%s' boards. The new rules on https://docs.qmk.fm/#/faq_build?id=linux-udev-rules offer better security with the same functionality.", bootloader)
+                    else:
+                        cli.log.warn("{bg_yellow}Missing udev rules for '%s' boards. You'll need to use `sudo` in order to flash them.", bootloader)
```

Comment:
I've been meaning to add the FAQ link to this warning as well.
What do you think?